packages/cli/src/zosfiles/-strings-/en.ts

```diff
@@ -40,6 +40,12 @@ export default {
                     EX1: "Create an empty C code PDS with default parameters"
                 }
             },
+            DATA_SET_LIKE: {
+                DESCRIPTION: "Create data sets for C code programming",
```

Comment:
This should be something like "Create data sets based on the properties of an existing data set"